backend/testUtils/testDb.ts

```diff
@@ -7,7 +7,7 @@ const mongo = new MongoMemoryServer();
 const mongoTest = {
   connect: async (): Promise<void> => {
     await mongo.start();
-    const uri = mongo.getUri();
+    const uri = await mongo.getUri();
```

Comment:
why was this modified for linting?

Reply:
oh, I'm not sure but it was giving an error if I didn't have it so I added it